qemu/tests/virtio_mem_numa_basic.py

```diff
@@ -0,0 +1,69 @@
+import time
+
+from virttest import error_context
+
+from virttest.utils_misc import normalize_data_size
+from virttest.utils_misc import NumaInfo
+from provider import virtio_mem_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Memory share and discard-data hotplug test
+    1) Boot guest with two virtio-mem devices
+    2) Check virtio-mem devices
+    3) Resize virtio-mem devices
+    4) Check virtio-mem devices
+    5) Resize virtio-mem devices to the maximum
+    6) Check virtio-mem devices
+    7) Resize virtio-mem devices to the minimum
+    8) Check virtio-mem devices
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    host_numa_node = NumaInfo()
+    node_list = host_numa_node.online_nodes_withmem
+    if len(node_list) < 2:
+        test.cancel("This host only has one NUMA node, skipping test...")
```

Comment:
Why do we need two numa on the host?
If I understand correctly, we just boot with two numa guests and are not bound to host numa respectively.
Manual testing can also be performed on a single numa host.

Reply:
That's true, sorry... now deleted. I took another numa auto case as example and I didn't pay too much attention to that part, I considered that it was ok... :confused: